src/animation/blend_state.cpp

```diff
@@ -2,6 +2,14 @@
 
 using namespace rive;
 
+BlendState::~BlendState()
+{
+	for (auto anim : m_Animations)
```

Comment:
This is a really good catch and makes me realize we may have a few more conditions like this.